data/repositories.toml

```diff
@@ -803,5 +803,6 @@ repositories = [
   'github.com/zulip/zulip',
   'github.com/ZupIT/charlescd',
   'github.com/ZupIT/ritchie-cli',
-  'github.com/ZupIT/ritchie-formulas'
+  'github.com/ZupIT/ritchie-formulas',
+  'github.com/restqa/restqa'
```

Comment:
Hey @olivierodo - Thanks for contributing.
Please add it in the lexicographic order